pypeit/core/datacube.py

```diff
@@ -571,84 +349,84 @@ def extract_standard_spec(stdcube, subpixel=20, method='boxcar'):
         box_gpm = flxscl > 1/3  # Good pixels are those where at least one-third of the standard star flux is measured
```

Comment:
Not sure I follow why you would every want to mask in this way. Boxcar is defined to be all the flux in the aperture.

Reply:
Gee... this was such a long time ago...

So, near the (spectral) edges there are masked pixels because each slice doesn't cover the same wavelength range. In this case, `flxscl` appears to be correcting for the flux that is missing (a value of 1 is no correction needed). You could also consider that the only good pixels are those with `flxscl = 1`, but it's been a while since I've looked at this.